tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Process/ProcessStartCommon.cs

```diff
@@ -42,15 +51,11 @@ internal static Scope CreateScope(string filename, IDictionary<string, string> e
 
             Scope scope = null;
 
+            // Arguments > ArgumentList : If Arguments is used, ArgumentList is ignored. ArgumentsList is only used if Arguments is an empty string.
```

Comment:
this comment could be the summary doc to the function `PopulateTags`, for everyone that would like to use it, to be able to just see this info on hover